main_routers/system_router.py

```
@@ -3220,15 +3223,36 @@ async def _fetch_meme_with_fallback(kw: str):
                                 aborted = True
                                 break
                             
-                            # 缓冲中剩余的文本作为首批内容
+                            # 缓冲中剩余的文本经由 pass_probe 逻辑输出
                             if cleaned.strip():
-                                full_text += cleaned
-                                await mgr.feed_tts_chunk(cleaned)
+                                combined = pass_probe + cleaned
+                                if '[PASS]' in combined.upper():
+                                    print(f"[{lanlan_name}] Phase 2 流式检测到 [PASS]，abort")
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)